core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -339,6 +343,25 @@ public ServerBuilder serviceLoggerPrefix(String serviceLoggerPrefix) {
         return this;
     }
 
+    /**
+     * Sets the format of this {@link Server}'s access log. The specified {@code accessLogFormat} would be
+     * parsed by {@link AccessLogWriters#custom(String)}.
+     */
+    public ServerBuilder accessLogFormat(String accessLogFormat) {
+        accessLogWriter = AccessLogWriters.custom(requireNonNull(accessLogFormat, "accessLogFormat"));
+        return this;
+    }
+
+    /**
+     * Sets an access log writer of this {@link Server}. {@link AccessLogWriters#disabled()} is used by default.
+     *
+     * @see AccessLogWriters to find pre-defined access log writers.
+     */
+    public ServerBuilder accessLogWriter(Consumer<? super RequestLog> accessLogWriter) {
+        this.accessLogWriter = requireNonNull(accessLogWriter, "accessLogWriter");
```

Comment:
I guess we can:

- declare `this.accessLogWriter` as `Consumer<RequestLog>`
- do unsafe cast here from `Consumer<? super RequestLog>` to `Consumer<RequestLog>`
- make `ServerConfig.accessLogWriter` a `Consumer<RequestLog>`

.. because:

- we are sure we will always feed a `RequestLog` to the `Consumer`.
- `Consumer` does not have any methods whose return type is affected by its type parameter.